src/ContextProvider/BankListContextProvider.php

```diff
@@ -17,6 +18,15 @@ public function __construct(
 
     public function provide(array $templateContext, TemplateBlock $templateBlock): array
     {
+        if (isset($templateContext['order'])) {
```

Comment:
Other option that I can see to do it is to split this Provider into 2 providers. One for checkout->complete and 2nd for this item selection on shop->order show->pay.
Cause on one the context contains `order` and for second not